### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by majovanilla**

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+  background: url(img/hero.jpg) no-repeat bottom / 100%;
+  height: 800px;
+  width: 100%;
+  padding-top: 270px;
+  text-align: center;
+  margin-bottom: 2px;
+}
+
+.small-image {
+  width: 360px;
```
Comment (Torres-ssf) [inline styles.css]: [REQUIRED]
Change the width to `25%`, this will make the 4 images take the full available width.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+  height: 200px;
+  float: left;
+  background-position: 50% 0%;
+  background-size: 100%;
```
Comment (Torres-ssf) [inline styles.css]: Add the line `background-repeat: no-repeat; `.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Nicely done @majovainilla :clap: :tada: 
I must say, your page looks really nice :smile: 
But before I give you :+1: , we need to make some minor changes on it.

**[REQUIRED]**
- [x]  There are some errors in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.

**[REQUIRED]**
- [x] We need to fix the errors found by the linter, that will improve the overall quality of your code and also gives a professional look in your repository.

**[REQUIRED]**
- [x] I left comments on your code to help you fix the size of the apple products promo sections

Please kindly make these changes and submit to code review again.
Happy coding :computer:

**Comment by samgaco** [comment]
#### Hi @majovainilla 

#### All the suggested changes have been well implemented, it's approved  :heavy_check_mark:  congratulations on finishing your first solo project. :clap: 

##### You may now merge to the master branch and submit the project completion form. :white_check_mark:

**Pull request merged**